<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="slider_time">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>671</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_stop">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>580</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_play">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>580</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>play/pause</string>
    </property>
   </widget>
   <widget class="QListView" name="lv_files">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>351</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_time">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>600</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>0:00</string>
    </property>
   </widget>
   <widget class="QListView" name="lv_text">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>310</y>
      <width>121</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="lv_stops">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>310</y>
      <width>91</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_stop_time">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>310</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_test_point">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>370</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Test</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add_stop_point">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>420</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_remove_stop_point">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>460</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_time_text">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>20</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_text_text">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>80</y>
      <width>281</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add_text">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>310</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_remove_text">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>360</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_open">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_title">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>80</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_album">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>190</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Title</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_album">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>170</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Album</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_save">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>88</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_test_text">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>480</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Text TEST</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_ip_adress">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>530</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;192.168.12.1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>976</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
